rest/verify/v2/docs/DefaultApi.md

```diff
@@ -167,12 +167,12 @@ Other parameters are passed through a pointer to a CreateChallengeParams struct
 
 Name | Type | Description
 ------------- | ------------- | -------------
-**AuthPayload** | **string** | Optional payload used to verify the Challenge upon creation. Only used with a Factor of type &#x60;totp&#x60; to carry the TOTP code that needs to be verified. For &#x60;TOTP&#x60; this value must be between 3 and 8 characters long.
-**DetailsFields** | **[]map[string]interface{}** | A list of objects that describe the Fields included in the Challenge. Each object contains the label and value of the field, the label can be up to 36 characters in length and the value can be up to 128 characters in length. Used when &#x60;factor_type&#x60; is &#x60;push&#x60;. There can be up to 20 details fields.
-**DetailsMessage** | **string** | Shown to the user when the push notification arrives. Required when &#x60;factor_type&#x60; is &#x60;push&#x60;. Can be up to 256 characters in length
+**AuthPayload** | **string** | Optional payload used to verify the Challenge upon creation. Only used with a Factor of type &#x60;totp&#x60; to carry the TOTP code that needs to be verified.
```

Comment:
Are you sure you are using the latest [twilio-oai specs](https://github.com/twilio/twilio-oai/tree/main/spec/yaml) for the generation? For example, [this shouldn't be changing. ](https://github.com/twilio/twilio-oai/blob/main/spec/yaml/twilio_verify_v2.yaml#L1556)

Reply:
Fixed